src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -61,6 +67,52 @@ protected function add_filters() {
 		add_filter( 'tec_tickets_commerce_admin_notices', [ $this, 'filter_admin_notices' ] );
 	}
 
+	/**
+	 * Process the async stripe webhook.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $order_id The order ID.
+	 *
+	 * @throws Exception If the action fails after too many retries.
+	 */
+	public function process_async_stripe_webhook( int $order_id ): void {
+		$order = tec_tc_get_order( $order_id );
+
+		if ( ! $order || ! $order instanceof WP_Post || ! $order->ID ) {
+			return;
+		}
+
+		$pending_webhooks = get_post_meta( $order->ID, '_tec_tickets_commerce_stripe_webhook_pending' );
+
+		// On multiple checkout completes, make sure we dont process the same webhook twice.
+		delete_post_meta( $order->ID, '_tec_tickets_commerce_stripe_webhook_pending' );
```

Comment:
Store this meta key somewhere. I like something Protected with an access public in the future if we ever need it.

Reply:
ill do a public const - thats equivalent to what you are saying i think.
write => no
read => yes